editor/header/tools/style.scss

```diff
@@ -106,6 +106,13 @@
 	margin-bottom: 0;
 }
 
+@media ( max-width: $break-large ) {
```

Comment:
You can use `@include break-large()` here.

Reply:
Updated to make it mobile first and moved styling to the component's style file.